src/ir_Fujitsu.cpp

```diff
@@ -0,0 +1,218 @@
+  uint8_t len = getCommandLength();
+  _irsend.sendFujitsuAC(remote_state, len);
+}
+
+uint8_t IRFujitsuAC::getCommandLength() {
+  uint8_t len = FUJITSU_AC_STATE_LENGTH;
+  if (remote_state[5] != 0xFE)
+    len = FUJITSU_AC_STATE_LENGTH_SHORT;
+  return len;
+}
+
+// Return a pointer to the internal state date of the remote.
+uint8_t* IRFujitsuAC::getRaw() {
+  remote_state[0] = 0x14;
+  remote_state[1] = 0x63;
+  remote_state[2] = 0x00;
+  remote_state[3] = 0x10;
+  remote_state[4] = 0x10;
+  bool fullCmd = false;
+  switch (_cmd) {
+    case FUJITSU_AC_CMD_TURN_OFF:
+      remote_state[5] = 0x02;
+      break;
+    case FUJITSU_AC_CMD_STEP_HORIZ:
+      remote_state[5] = 0x79;
+      break;
+    case FUJITSU_AC_CMD_STEP_VERT:
+      remote_state[5] = 0x6C;
+      break;
+    default:
+      remote_state[5] = 0xFE;
+      fullCmd = true;
+      break;
+  }
+  if (fullCmd) {
+    remote_state[6] = 0x09;
+    remote_state[7] = 0x30;
+    uint8_t tempByte = _temp - 16;
+    remote_state[8] = (_cmd == FUJITSU_AC_CMD_TURN_ON) | (tempByte << 4);
+    remote_state[9] = _mode | 0 << 4;  // timer off
+    remote_state[10] = _fanSpeed | _swingMode << 4;
+    remote_state[11] = 0;  // timerOff values
+    remote_state[12] = 0;  // timerOff/on values
+    remote_state[13] = 0;  // timerOn values
+    remote_state[14] = 0x20;
+    // Checksum is such that all bytes 8 to 16 (NB 16 is the checksum itself)
+    // sum to zero
+    uint8_t checksum = 0;
+    for (uint8_t i = 7 ; i < 15; ++i) {
+      checksum += remote_state[i];
+    }
+    remote_state[15] = 0 - checksum;
+  } else {
+    // For the short codes, byte 7 is the inverse of 6
+    remote_state[6] = ~remote_state[5];
+    for (uint8_t i = 7; i < FUJITSU_AC_STATE_LENGTH; ++i) {
+      remote_state[i] = 0;
+    }
+  }
+  return remote_state;
+}
+
+// Set the requested power state of the A/C to off.
+void IRFujitsuAC::off() {
+  _cmd = FUJITSU_AC_CMD_TURN_OFF;
+}
+
+void IRFujitsuAC::stepHoriz() {
+  _cmd = FUJITSU_AC_CMD_STEP_HORIZ;
+}
+
+void IRFujitsuAC::stepVert() {
+  _cmd = FUJITSU_AC_CMD_STEP_VERT;
+}
+
+// Set the requested command of the A/C.
+void IRFujitsuAC::setCmd(uint8_t cmd) {
+  switch (cmd) {
+    case FUJITSU_AC_CMD_TURN_OFF:
+    case FUJITSU_AC_CMD_TURN_ON:
+    case FUJITSU_AC_CMD_STAY_ON:
+    case FUJITSU_AC_CMD_STEP_HORIZ:
+    case FUJITSU_AC_CMD_STEP_VERT:
+      break;
+    default:
+      cmd = FUJITSU_AC_CMD_STAY_ON;
+      break;
+  }
+  _cmd = cmd;
+}
+
+uint8_t IRFujitsuAC::getCmd() {
+  return _cmd;
+}
+
+// Set the temp. in deg C
+void IRFujitsuAC::setTemp(uint8_t temp) {
+  temp = std::max((uint8_t) FUJITSU_AC_MIN_TEMP, temp);
+  temp = std::min((uint8_t) FUJITSU_AC_MAX_TEMP, temp);
+  _temp = temp;
+}
+
+uint8_t IRFujitsuAC::getTemp() {
+  return _temp;
+}
+
+// Set the speed of the fan
+void IRFujitsuAC::setFanSpeed(uint8_t fanSpeed) {
+  if (fanSpeed > FUJITSU_AC_FAN_QUIET)
+    fanSpeed = FUJITSU_AC_FAN_QUIET;  // Set the fan to maximum if out of range.
```

Comment:
Suggestion:
I'm assuming the quiet value is numerically larger than the highest normal fan value. i.e. maximum device fan speed etc.
You might want to set any value larger than we know of to the maximum device fan value, not to just the highest value. i.e. quiet etc.